src/modules/form/util/formUtil.ts

```diff
@@ -756,14 +756,17 @@ export const formValueToGqlValue = (
     if (typeof date === 'string') {
       date = parseHmisDateString(value);
     }
-    if (date instanceof Date) return formatDateForGql(date) || undefined;
-    // This isn't parseable/formattable into a date, return undefined to ignore it
-    return undefined;
+    if (date instanceof Date) return formatDateForGql(date) || value;
+    // This isn't parseable/formattable into a date, so it may cause a backend validation error.
+    // Still, return it, so that it doesn't get swallowed without the user's knowledge
+    // TODO(#6713) - Frontend should prevent submission of invalid dates
```

Comment:
This doesn't feel like quite the right place for a TODO, since I don't think this util function logic will _change_ with 6713, but maybe I'm wrong about that

Reply:
You're right, I can add a note to the ticket instead